t/mirror.t

```diff
@@ -15,11 +15,11 @@ use CommonTests;
 
 my $cwd         = getcwd;
 my $test_mirror = "file:///$cwd/t/CPAN";
-my $cache       = File::Temp->newdir;
 my $mailrc      = "01mailrc.txt";
 my $packages    = "02packages.details.txt";
 
 sub new_mirror_index {
+    my $cache = File::Temp->newdir;
```

Comment:
As in the last test file, you should now check that after `new_ok`, that `$index->cache eq $cache`.